src/output-json-alert.c

```diff
@@ -1059,6 +1079,13 @@ static void JsonAlertLogSetupMetadata(AlertJsonOutputCtx *json_output_ctx,
         json_output_ctx->payload_buffer_size = payload_buffer_size;
     }
 
+    /* action is enabled by default */
+    flags |= LOG_JSON_ACTION;
+    SetFlag(conf, "action", LOG_JSON_ACTION, &flags);
```

Comment:
I think that this change is triggering the debug validation checks. Investigating....

Reply:
It is that, indeed, as conf can be NULL here.